### PR TITLE
On Prem Blog

### PR DESCRIPTION
**Pull request by shyamvalsan**
On Prem Blog

### CODE REVIEW

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+### Control & Customization
+Have specific monitoring needs? The on-prem version allows you to tweak and tailor your monitoring parameters, ensuring that the system aligns perfectly with your unique requirements. From custom dashboards to specialized alerts, take control of your monitoring landscape like never before.
+
+### Cost-Effectiveness
+While cloud solutions might seem economical initially, the costs can escalate with data volume and system complexity. Netdata Cloud On-Prem offers an economically sound alternative that can scale efficiently within your own premises.
+
+### Enterprise-Grade Support
+We understand that monitoring isn't the primary focus of your business or your team, which is why we offer professional services to help you get the most out of Netdata Cloud On-Prem. From setup to ongoing maintenance, our enterprise-grade support ensures that you can focus on what you do best.
+
+## Who should consider switching to Netdata Cloud on prem?
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:52]: ### Who should consider switching to Netdata Cloud on prem?

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Large Enterprises
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:54]: **Large Enterprises**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+With infinite horizontal scalability, manage vast arrays of nodes without breaking a sweat.
+
+### Governmental Organizations
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:57]: **Governmental Organizations**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+When you have security protocols that are non-negotiable, on-prem monitoring is your go-to.
+
+### Critical Infrastructure
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:60]: **Critical Infrastructure**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Ensure real-time, high-speed data processing, and immediate action, crucial for mission-sensitive operations.
+
+### Organizations with large, diverse and dynamic infrastructure
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:63]: **Organizations with large, diverse and dynamic infrastructure**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+The Netdata Cloud On-Prem package is engineered to handle tens of thousands of nodes, making it ideal for companies with expansive network architectures.
+
+### Teams that require Enterprise-grade support
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:66]: **Teams that require Enterprise-grade support**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+Leverage our professional services to relieve the monitoring setup burden and focus on your core business operations.
+
+## Technical Specifications
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:69]: #### Technical Specifications

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+Netdata Cloud is a highly-available, scalable, microservices environment, based on Kubernetes. Netdata Cloud automatically scales to support an infinite number of nodes. The installation we run today, supports hundreds of thousands of concurrently online nodes, with tens of thousands of nodes being added, removed and modified daily.
+
+Netdata Cloud On-Prem runs on a Kubernetes environment and comes embedded with all necessary services for a standalone operation. It requires the following open-source components:
+- Kubernetes: The backbone for installing Netdata Cloud.
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:74]: **Kubernetes:**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+- PostgreSQL: Our main relational database.
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:75]: **PostgreSQL:**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+- Elasticsearch: For audit logs, topology changes, and alert triggers.
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:76]: **Elasticsearch:**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+- Pulsar: Our message broker for microservices communication.
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:77]: **Pulsar:**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+- Email Service: Required for email notifications (SMTP or Sendgrid).
```
Comment (Aliki92) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:78]: **Email Service:**

File: blog/2023-09-26-netdata-cloud-on-prem/index.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+tags: [netdata, netdata cloud, on prem, on premises, infrastructure monitoring, monitoring]
+keywords: [netdata, netdata cloud, on prem, on premises, infrastructure monitoring, monitoring]
+image: https://github.com/netdata/blog/assets/24860547/7e503687-0472-44b0-bf6f-0bff40aeb517
+---
+
+![netdata-cloud-on-prem](https://github.com/netdata/blog/assets/24860547/7e503687-0472-44b0-bf6f-0bff40aeb517)
+
+
+
+Discover the next level of on premise infrastructure monitoring with Netdata Cloud On-Prem. Learn how to efficiently monitor your infrastructure & lower your total cost ownership with full security and privacy compliance guaranteed
```
Comment (hugovalente-pm) [inline blog/2023-09-26-netdata-cloud-on-prem/index.md:14]:
```suggestion
Discover the next level of on premise infrastructure monitoring with Netdata Cloud On-Prem. Learn how to efficiently monitor your infrastructure & lower your total cost ownership with full security and privacy compliance guaranteed.
```

if this isn't deleted it misses the `.`

**Review by hugovalente-pm — APPROVED** [review]

**Pull request merged**